target/linux/ar71xx/image/generic.mk

```diff
@@ -161,6 +161,17 @@ define Device/gl-mifi
 endef
 TARGET_DEVICES += gl-mifi
 
+define Device/lima
+  DEVICE_TITLE := Lima board from 8Devices
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2
+  BOARDNAME = LIMA
+  BLOCKSIZE = 256k
+  IMAGE_SIZE = 15616k
+  MTDPARTS = spi0.0:256k(u-boot)ro,256k(u-boot-env)ro,256k(art)ro,-(firmware)
+  IMAGE/sysupgrade.bin = append-kernel $$(BLOCKSIZE) | append-rootfs | pad-rootfs | check-size $$$$(IMAGE_SIZE)
```

Comment:
This line doesn't look correct for me (this part: `append-kernel $$(BLOCKSIZE)` is wrong).
I think you can just drop the whole line and relay on [default definition](https://github.com/lede-project/source/blob/master/target/linux/ar71xx/image/Makefile#L110).